tests/security/cc/netfilter.pm

```diff
@@ -0,0 +1,147 @@
+
+    # configure the network
+    my $data = get_test_suite_data();
+
+    my $role = get_required_var('ROLE');
+    foreach my $key (keys %{$data->{$role}}) {
+        my $n       = $data->{$role}->{$key};
+        my $netcard = $n->{netcard};
+        my $dev     = $netcard;
+        assert_script_run("ip link add link eth0 address $n->{mac_add} $netcard type macvlan");
+
+        # Network Bridge setting for Target of Evaluation(TOE)
+        if ($role eq 'client' && $key eq 'second_interface') {
+            assert_script_run('brctl addbr toebr');
+            assert_script_run("brctl addif toebr $netcard");
+            assert_script_run('brctl setageing toebr 3600');
+            $dev = 'toebr';
+        }
+
+        assert_script_run("ip addr add $n->{ipv4} dev $dev");
+        assert_script_run("ip addr add $n->{ipv6} dev $dev");
+        assert_script_run("ip link set $netcard up");
+        assert_script_run('ip link set toebr up') if ($role eq 'client' && $key eq 'second_interface');
+        assert_script_run("ip -6 route add $n->{route} dev $dev");
+    }
+
+    # start lblnet_tst_server
+    my $cmd        = "$audit_test::testdir$audit_test::testfile_tar/audit-test/utils/network-server/lblnet_tst_server";
+    my $lblnet_pid = background_script_run($cmd);
+
+    my $result = 'ok';
+    if ($role eq 'server') {
+        mutex_create('NETFILTER_SERVER_READY');
+        wait_for_children;
+    } else {
+        mutex_wait('NETFILTER_SERVER_READY');
+
+        # export the variables
+        my $client_first  = $data->{client}->{first_interface};
+        my $client_second = $data->{client}->{second_interface};
+        my $server_first  = $data->{server}->{first_interface};
+        my $server_second = $data->{server}->{second_interface};
+
+        # Deal with ipv4 address: change 192.168.0.1/24 to 192.168.0.1
+        $client_first->{ipv4}  =~ s/\/.*//g;
+        $client_second->{ipv4} =~ s/\/.*//g;
+        $server_first->{ipv4}  =~ s/\/.*//g;
+        $server_second->{ipv4} =~ s/\/.*//g;
+
+        script_run("export PASSWD=$testapi::password");
+        script_run("export LOCAL_DEV=$client_first->{netcard}\@eth0");
+        script_run("export LOCAL_SEC_DEV=$client_second->{netcard}\@eth0");
+        script_run("export LOCAL_SEC_MAC=$client_second->{mac_add}");
+        script_run("export LOCAL_IPV4=$client_first->{ipv4}");
+        script_run("export LOCAL_IPV6=$client_first->{ipv6}");
+        script_run("export LOCAL_SEC_IPV4=$client_second->{ipv4}");
+        script_run("export LOCAL_SEC_IPV6=$client_second->{ipv6}");
+
+        script_run("export LBLNET_SVR_IPV4=$server_first->{ipv4}");
+        script_run("export LBLNET_SVR_IPV6=$server_first->{ipv6}");
+        script_run("export SECNET_SVR_IPV4=$server_second->{ipv4}");
+        script_run("export SECNET_SVR_IPV6=$server_second->{ipv6}");
+        script_run("export SECNET_SVR_MAC=$server_second->{mac_add}");
+
+        script_run('export BRIDGE_FILTER=toebr');
```

Comment:
Since there are so many variables need to be exported I suggest to use one line to save time, e.g.,
`export PASSWD=$pw LOCAL_DEV=$local_dev ...` and use `assert_script_run` instead of `script_run`